_health-care/_js/_form.js

```diff
@@ -1,21 +1,47 @@
 var HealthApp = (function() {
   "use strict";
 
-  var identityTransform = function(formData, arg) { return arg; };
-  var extractTransform = function(formData, arg) { return formData[arg]; };
+  function identityTransform(formData, arg) { return arg; }
+  function extractTransform(formData, arg) { return formData[arg]; }
```

Comment:
And by later today...I meant right now.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Closures
http://stackoverflow.com/questions/4071292/may-function-declarations-appear-inside-statements-in-javascript

So I'm wrong in 2 counts.
  1. Normal FunctionDeclarations (function iHaveAName() {}) do indeed create a closure if necessary.
  2. FunctionDeclarations are not allowed inside a block (anything surrounded by a {}); only Function Expressions are allowed. This is rooted in ECMAScript spec 5 so it's definitely authoritative. The fact that it works is actually a commonly implemented extension to JS.

Oops. I will fix in another PR cause I have a meeting to prep for right now.